units/meson.build

```diff
@@ -267,6 +267,15 @@ units = [
           'file' : 'systemd-boot-update.service',
           'conditions' : ['ENABLE_BOOTLOADER'],
         },
+        {
+          'file' : 'systemd-bootctl@.service.in',
+          'conditions' : ['ENABLE_BOOTLOADER'],
```

Comment:
Didn't we change bootctl to always be enabled recently? Can you double check?

Reply:
That was primarily for image builds. Not sure if it makes sense for the associated service to always be installed.

Reply:
So dedb925eafee8214ae565b861dfacfc02085f158 says this:

> bootctl is rather useful to have, even if on a system without UEFI, as it has a number of verbs that are unrelated to UEFI (e.g kernel-identify), and more importantly, it supports --root to operate on directory trees (which could be intended to be deployed on UEFI) so let's make sure we always build it.

Of the mentioned functionality the varlink service doesn't actually expose anything. hence for now I think we shouldn't drop the conditionalization...